dockerfiles/Dockerfile.sandbox

```diff
@@ -18,9 +18,14 @@ FROM tiangolo/uwsgi-nginx-flask:python3.10
 # Install dependencies required for Lean 4, pypy3, and other tools
 RUN apt-get update && \
     apt-get install -y curl git net-tools bzip2 build-essential libseccomp-dev && \
-    curl -L https://downloads.python.org/pypy/pypy3.10-v7.3.17-linux64.tar.bz2 -o /tmp/pypy.tar.bz2 && \
+    case "$TARGETARCH" in \
```

Comment:
Does this need to redefined after the first `FROM` call?

As it recommends here: https://docs.docker.com/build/building/variables/#multi-platform-build-arguments

Reply:
So the suggestion is to add this before `RUN apt-get`
```
ARG=TARGETARCH
```
?